src/components/Card.js

```diff
@@ -1,33 +1,52 @@
-import React from "react";
-import styled from "styled-components";
+import styled from 'styled-components';
+import React, { useState, useEffect } from 'react';
 
 const Card = () => {
+  const [characters, setCharacters] = useState([]);
```

Comment:
The fetching itself is good, but the fetching logic belongs into a Conatiner COmponent (e.g. the App.js or a CardList component) component. The Card component should only render one card.
in the container Component there should be some logic similar to:

{characters.map(character => <Card character={character} ... />)}